tools/webpack-core/src/utils/resolve-recurse/resolve.js

```diff
@@ -0,0 +1,144 @@
+/*---------------------------------------------------------------------------------------------

```

Comment:
This file is from that package I mentioned awhile back that was extremely old and no longer supported. Its mostly the original file with the exception of a few of my changes in it as well. I'm not sure if I did the licenses correctly though, could someone comment on that?

Reply:
@calebmshafer - you're probably the vendoring expert

Reply:
We shouldn't add our copyright header to it, if we did not originally write it so I'd remove that and leave the file as it was in the original source.

Reply:
Re-thinking this, lets leave our copyright header and then just add a line of attribution below it. Along the lines of

```
Adapted from resolve.js, https://github.com/brandonhorst/node-resolve-recurse/blob/master/lib/resolve.js
```

Reply:
Changed

Reply:
LGTM